### DISCUSSION TITLE
Sesh v2, first beta release!

### DISCUSSION THREAD

**Discussion by joshmedeski** [joshmedeski/sesh#138] [General]
Sesh has been completely rewritten over the last few months! I'm not feature complete yet, but I'm ready for beta tests to help catch bugs and prepare for release.

Here is what's here:
- Completely rewritten codebase using the dependency injection pattern
- Simplified and more predictable and organized logic
- Automated package mocks using Mockery

Features completed so far:
- `sesh list`
- `sesh connect`

Features to complete:
- `sesh clone`
- Icon support

What's changed?
- Dropped `startup_script` in favor of `startup_command`
```diff
-startup_script = "~/.config/sesh/scripts/node_dev"
+startup_command = "~/.config/sesh/scripts/node_dev"
```

**Note:** You'll need to make sure your script files are executable.

Please submit an GitHub issue or go to the linked discussion with any bugs, unexpected behavior, or requests as I finalize the v2 release, thanks!

<hr /><em>This discussion was created from the release <a href='https://github.com/joshmedeski/sesh/releases/tag/v2.0.0-beta.1'>Sesh v2, first beta release!</a>.</em>

**Comment by H4ckint0sh** [comment]
How to install V2 with brew? There is no HEAD formula?

**Reply by joshmedeski** [reply]
It is officially released! So you can update it with brew
